### PR TITLE
[RW-7885, PF-1376] update from bytes to MiB

### PR DESCRIPTION
**Pull request by jaycarlton**
I had the base unit wrong; it was supposed to be in MiB, so I was off by a factor of a million or so.

Add a new config, `vp_sc_v9`.

### CODE REVIEW

**Review by zloery — APPROVED** [review]

File: src/main/resources/config/resource_schema.yaml [hunk 1]
```diff
@@ -194,9 +194,9 @@ components:
             and there's no way to generate Optional types.
           type: boolean
           default: false
-        bigQueryDailyUsageQuotaOverrideValueBytes:
+        bigQueryDailyUsageQuotaOverrideValueMebibytes:
```
Comment (zloery) [inline src/main/resources/config/resource_schema.yaml:197]: I think changing this key will break the existing v8 config. For compatibility, it seems like you might need to introduce a new key -> deprecate and delete the v8 pool -> remove the old key.

Alternatively I wonder if you could edit the key name in the v8 config? I'm not sure if RBS lets you do that

Comment (jaycarlton) [reply src/main/resources/config/resource_schema.yaml:197]: I'm trying the latter suggestion, b/c it fails a test if I don't. There's only one current schema, so when it changes, all the configs need to adhere to it.

Another option would be to add a new key set to the old value and leave the v8 schema intact, then remove it once we've removed v8. It's more churn, so if the naive way works, I'm going to stick with it.

File: src/main/resources/config/dev/resource-config/vpc_sc_v8.yml [hunk 2]
```diff
@@ -37,4 +37,4 @@ gcpProjectConfig:
   serviceUsage:
     bigQuery:
       overrideBigQueryDailyUsageQuota: true
-      bigQueryDailyUsageQuotaOverrideValueBytes: 43980465111040 # 40 TB
+      bigQueryDailyUsageQuotaOverrideValueMebibytes: 43980465111040 # 40 TB
```
Comment (calbach) [inline src/main/resources/config/dev/resource-config/vpc_sc_v8.yml:40]: The 40TB comment is now outdated - but also, should this number adjusted back down to actually be 40 TB?

Note: if this change is possible, do we need the v9 pool at all? Personally I don't care if the next ~300 projects from v8 have an incorrect BigQuery quota - we'll probably go through those in a week or so. As long as one project gets created with this new setting, we can verify the fix via cloud console.

Comment (jaycarlton) [reply src/main/resources/config/dev/resource-config/vpc_sc_v8.yml:40]: My thought process was that we only need to change the v8 pool config to make it loadable under the new schema. I don't know if it's safe to change the value. I suppose we save a round trip to the other repo to point to v9 if we're sure changing v8 will work.

Zach's approach might be the most fool-proof, and would preserve history in the pools.

Comment (calbach) [reply src/main/resources/config/dev/resource-config/vpc_sc_v8.yml:40]: I defer to you and @zloery, I'm fine with either approach and don't have enough RBS experience to say whether changing v8 would work. My only request is either fix the comment or fix the value here before merging.

**Review by calbach — APPROVED** [review]

File: src/main/resources/config/resource_schema.yaml (hunk 1 above)
Comment (jaycarlton) [reply src/main/resources/config/resource_schema.yaml:197]: Went with the safe route, keeping the old key but deprecating it.

**Pull request merged**